CHANGELOG.rst

```diff
@@ -13,6 +13,9 @@ Changes:
 
 - Don't overwrite ``__name__`` with ``__qualname__`` for ``attr.s(slots=True)`` classes.
   `#99 <https://github.com/hynek/attrs/issues/99>`_
+- Allow for a ``__post_init__`` method that, if defined, will get executed at
```

Comment:
We use semantic new lines in prose. Simply make each sentence a new line please.

Also please add a link to this PR like in the entry before.